### DISCUSSION TITLE
Allow disabling notifications from forked repositories of other people

### DISCUSSION THREAD

**Discussion by craftzdog** [community/community#16137] [Other Feature Feedback, Questions, & Ideas]
I have [a public repository](https://github.com/craftzdog/craftzdog-homepage) integrated with Vercel.
Vercel posts comment to the commits to notify deployment results like this one:

https://github.com/craftzdog/craftzdog-homepage/commit/5d47dfb73bc1974e51a0ca3be1e6a415636a5168#commitcomment-71441623

But I keep getting such notifications from the forked repositories of other people as well, which is annoying.
Those repos are even private, so I don't have any control over them.

![CleanShot 2022-05-06 at 17 54 08@2x](https://user-images.githubusercontent.com/1332805/167100249-a1751569-099f-4561-83a5-5d5741cb7f3d.png)

How can I disable receiving those notifications? I'm not talking about the 'unsubscribe' link in the email.
I want to suppress *all* the notifications from the forked reops.
Vercel answered that they can't control that.

**Comment by ThePhoDit** [comment]
Same is happening for me!

**Comment by frectonz** [comment]
Same here.
Why is this behavior the default?
Is there some way this feature is useful?

**Comment by mscdex** [comment]
Yeah, this is very annoying. Can we get an option to disable all notifications from forked repos? While we're at it, can we also get an option to disable notifications resulting from "@" mentions in commit messages?

**Comment by xream** [comment]
Any updates?
